src/vector/index.js

```diff
@@ -313,9 +313,13 @@ async function loadApp() {
                 // FIXME: we should probably block loading the app or even
                 // showing a spinner until the theme is loaded, to avoid
                 // flashes of unstyled content.
-                a.onload = () => {
+                if (typeof InstallTrigger !== 'undefined') {
```

Comment:
Hmm. please comment this kind of thing (imagine how confused you'd be if you were reading this file and found it looking at something called `InstallTrigger`...)

Also, is there not a way to do this based on whether the CSS is loaded or not? I'm thinking something like https://gist.github.com/cvan/8a188df72a95a35888b70e5fda80450d although I tested that and Chrome still thinks the stylesheet has rues before `onload` is fired. How does Chrome fail if you run the Tinter without the onload? Can we not detect that failure and run the Tinter conditionally on that?

Reply:
I sure can add a comment. Sorry for having missed to do that.